src/main/kotlin/org/malachite/estella/security/Authorities.kt

```diff
@@ -0,0 +1,9 @@
+package org.malachite.estella.security
+
+import org.springframework.security.core.GrantedAuthority
+
+enum class Authority : GrantedAuthority {
+    job_seeker, hr, organization;
```

Comment:
Jak to w końcu robimy? Rozróżniamy trzy role, a potem w każdym servicie patrzymy, czy ta rola jest ok dk wykonania akcji, czy ma id odpowiednie itp? Nie da się już tego tutaj zrobić wtedy?